common/djangoapps/util/views.py

```diff
@@ -84,6 +84,7 @@ def inner(request, *args, **kwargs):
                     # In debug mode let django process the 500 errors and display debug info for the developer
                     raise
                 else:
+                    log.exception("Error in django view.")
```

Comment:
It is going to print complete stack trace along with exception message. Here is what i get in the logs while testing it on sandbox.
```
ERROR [certstest  24221] [views.py:87] - Error in django view.
Traceback (most recent call last):
  File "/edx/app/edxapp/edx-platform/common/djangoapps/util/views.py", line 81, in inner
    return func(request, *args, **kwargs)
  File "/edx/app/edxapp/edx-platform/lms/djangoapps/certificates/views/webview.py", line 476, in render_html_view
    return HttpResponse(template.render(context))
  File "/edx/app/edxapp/edx-platform/common/djangoapps/edxmako/template.py", line 62, in render
    return super(Template, self).render_unicode(**context_dictionary)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/mako/template.py", line 454, in render_unicode
    as_unicode=True)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/mako/runtime.py", line 829, in _render
    **_kwargs_for_callable(callable_, data))
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/mako/runtime.py", line 864, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/mako/runtime.py", line 890, in _exec_template
    callable_(context, *args, **kwargs)
  File "memory:0xa9ea510", line 183, in render_body
AttributeError: 'Undefined' object has no attribute 'url'
```